### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by andp-1202**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/43558

## WHAT (optional)
logout user

## Evidence (Screenshot or Video)
Ảnh chụp Màn hình 2021-11-30 lúc 14 37 51

### CODE REVIEW

**Comment by andp-1202** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def log_out
+    session.delete :user_id
```
Comment (vanvtt-0952) [inline app/helpers/sessions_helper.rb:7]: ko set `@current_user` k set vvề nil ah cu

**Comment by andp-1202** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**